build.gradle.kts

```diff
@@ -122,61 +120,57 @@ java {
 	}
 }
 
-signing {
-	isRequired = !isFork() && isAction()
-	sign(publishing.publications)
-}
-
-nexusPublishing {
-	this.repositories {
-		sonatype {
-			username.set(findProperty("ossrhUsername") as String?)
-			password.set(findProperty("ossrhPassword") as String?)
+mavenPublishing {
+	coordinates(project.group.toString(), "mockbukkit-v${property("paper.api.version")}", project.version.toString())
+
+	pom {
+		description.set("MockBukkit is a mocking framework for bukkit to allow the easy unit testing of Bukkit plugins.")
+		name.set("MockBukkit-v${property("paper.api.version")}")
+		url.set("https://github.com/MockBukkit/MockBukkit")
+		licenses {
+			license {
+				name.set("MIT License")
+				url.set("https://github.com/MockBukkit/MockBukkit/blob/v${property("paper.api.version")}/LICENSE")
+			}
 		}
-	}
-}
-
-publishing {
-	publications {
-		create<MavenPublication>("maven") {
-			artifactId = "mockbukkit-v${property("paper.api.version")}"
-			from(components.getByName("java"))
-			pom {
-				name.set("MockBukkit-v${property("paper.api.version")}")
-				description.set("MockBukkit is a mocking framework for bukkit to allow the easy unit testing of Bukkit plugins.")
-				url.set("https://github.com/MockBukkit/MockBukkit")
-				scm {
-					connection.set("scm:git:git://github.com/MockBukkit/MockBukkit.git")
-					developerConnection.set("scm:git:ssh://github.com:MockBukkit/MockBukkit.git")
-					url.set("https://github.com/MockBukkit/MockBukkit/tree/v${property("paper.api.version")}")
-				}
-				licenses {
-					license {
-						name.set("MIT License")
-						url.set("https://github.com/MockBukkit/MockBukkit/blob/v${property("paper.api.version")}/LICENSE")
-					}
-				}
-				developers {
-					developer {
-						id.set("seeseemelk")
-						name.set("Sebastiaan de Schaetzen")
-						email.set("sebastiaan.de.schaetzen@gmail.com")
-					}
-					developer {
-						id.set("thebusybiscuit")
-						name.set("TheBusyBiscuit")
-					}
-					developer {
-						id.set("insprill")
-						name.set("Pierce Thompson")
-					}
-					developer {
-						id.set("thelooter")
-						name.set("Eve Kolb")
-					}
-				}
+		developers {
+			developer {
+				id.set("seeseemelk")
+				name.set("Sebastiaan de Schaetzen")
+				email.set("sebastiaan.de.schaetzen@gmail.com")
+			}
+			developer {
+				id.set("thebusybiscuit")
+				name.set("TheBusyBiscuit")
+			}
+			developer {
+				id.set("insprill")
+				name.set("Pierce Thompson")
+			}
```

Comment:
@Insprill do you want your email in here?

Reply:
Sure